### DISCUSSION TITLE
Bar chart data overflows

### DISCUSSION THREAD

**Discussion by ataldev** [mantinedev/mantine#6482] [Q&A] [answered]
Hi,
In some cases, after the data is fetched from an API, the chart overflows and something like this is displayed:
<img width="815" alt="Screenshot 2024-07-02 at 11 56 52" src="https://github.com/mantinedev/mantine/assets/62022057/da342a95-d44c-43b1-bdf6-b072d8bd2b52">
See how the templates count is 11 but the limit for the views is still in 8. 
What can I do to solve this?

**Comment by rtivital** [comment]
Provide a sandbox with a minimal reproduction

**Reply by ataldev** [reply]
https://codesandbox.io/p/sandbox/recursing-pine-54gw7f

**Reply by rtivital** [reply, marked as answer]
https://codesandbox.io/p/sandbox/throbbing-river-rdsmtm?file=%2Fsrc%2FApp.js%3A88%2C41

**Reply by ataldev** [reply]
Thanks.
